lib/Params.ml

```diff
@@ -294,7 +294,9 @@ let get_if_then_else (c : Conf.t) ~first ~last ~parens ~parens_bch
     match xcond with
     | Some xcnd ->
         hvbox
-          (if parens then -2 else 0)
+          ( match imd with
+          | `No -> if parens then -1 else 0
```

Comment:
What do you think about matching with parens to simplify the logic?

```
match parens, imd with
| false, _ -> 0
| true, `No -> -1
| true, ... -> -2
```

(extracting a local variable for level might improve this, too)

Reply:
indeed, done! The whole function would need rewriting anyway, we need to simplify the formatting of if-then-else expressions and remove some of the supported styles!